yazi-plugin/preset/components/folder.lua

```diff
@@ -21,6 +21,8 @@ function Folder:linemode(area, files)
 			spans[#spans + 1] = ui.Span(time and os.date("%y-%m-%d %H:%M", time // 1) or "")
 		elseif mode == "permissions" then
 			spans[#spans + 1] = ui.Span(f.cha:permissions() or "")
+		elseif mode == "owner" then
+			spans[#spans + 1] = ui.Span(f.cha:owner() or "")
```

Comment:
Please use the existing `user_name()` instead, https://yazi-rs.github.io/docs/plugins/utils#ya.user_name